frontend/packages/container-security/src/components/summary.tsx

```diff
@@ -77,19 +107,19 @@ export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = ({
                             title={priority.title}
                           />
                           &nbsp;
-                          {_.uniqBy(vulnsFor(priority.value), 'metadata.name').length}{' '}
-                          {priority.title}
+                          {t('container-security~{{vulnImageCount}} {{priorityTitle}}', {
+                            vulnImageCount: _.uniqBy(vulnsFor(priority.value), 'metadata.name')
+                              .length,
+                            priorityTitle: priority.title,
+                          })}
```

Comment:
Translation is not needed here. It won't have any effect.

Reply:
this adds translation to the **Vulnerability Container Images** list. The list has items with the count of images with priorities like `1 High`. Do we not need translation for this? I have attached images for reference.
**Without translation**
![Screenshot from 2020-11-02 18-40-28](https://user-images.githubusercontent.com/38663217/97873042-0f30d080-1d3d-11eb-9a15-57d6f3eb404a.png)
**With Translation**
![Screenshot from 2020-11-02 18-51-39](https://user-images.githubusercontent.com/38663217/97873080-1d7eec80-1d3d-11eb-898b-fcd7b090bcaa.png)